src/main/scala/org/reactivecouchbase/rs/scaladsl/Bucket.scala

```diff
@@ -56,6 +57,8 @@ class Bucket(config: BucketConfig, onStop: () => Unit) {
     (_bucket, _bucket.async(), _bucketManager, Future.successful(_bucket.async()))
   }
 
+  private val durationToExpiry = (expiration:Duration) => ((if (expiration < 30.days) 0L else System.currentTimeMillis / 1000L) + expiration.toSeconds).toInt
```

Comment:
Anyway, I'll merge the PR and update the code. 

Thanks for your contribution @javolek